src/applications/static-pages/facilities/FacilityListWidget.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+
+export default class FacilityListWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+      parsedFacilities: JSON.parse(this.props.facilities),
+    };
+  }
+
+  componentDidMount() {
+    const facilityIds = Object.keys(this.state.parsedFacilities);
+    apiRequest(
+      `/facilities/va?ids=${facilityIds}`,
+      null,
+      this.handleFacilitiesSuccess,
+      this.handleFacilitiesError,
+    );
+  }
+
+  handleFacilitiesSuccess = facilities => {
+    this.setState({
+      loading: false,
+      facilities: facilities.data,
+    });
+  };
+
+  handleFacilitiesError = () => {
+    this.setState({ error: true });
+  };
+
+  facilitiesList = facilities => {
+    const sortedFacilities = facilities.sort((a, b) => {
+      const aName = a.attributes.name;
+      const bName = b.attributes.name;
+      if (aName < bName) {
+        return -1;
+      }
+
+      if (aName > bName) {
+        return 1;
+      }
+
+      return 0;
+    });
+
+    return sortedFacilities.map(facility => {
```

Comment:
I think I would return `sortedFacilities` from this function and move this map into the jsx in the render method. It's a little bit easier to read if the primary jsx output is in the render method of a component.